src/main/java/net/sf/jabref/gui/entryeditor/EntryEditorTab.java

```diff
@@ -155,7 +155,7 @@ private void setupPanel(JabRefFrame frame, BasePanel bPanel, boolean addKeyField
             } else {
                 String prompt = "";
                 if (field.equals(FieldName.AUTHOR) || field.equals(FieldName.EDITOR)) {
-                    prompt = String.format("%1$s and %1$s and others", Localization.lang("Firstname Lastname"));
+                    prompt = String.format("%1$s " + Localization.lang("and others") , Localization.lang("Firstname Lastname"));
```

Comment:
This is not the same. `Firstname Lastname and Firstname Lastname and others` vs `Firstname Lastname and others`, also `and` and `and others` should not be translated. See https://github.com/JabRef/jabref/pull/2147#discussion_r82714423, https://github.com/JabRef/jabref/pull/2147#discussion_r82861252
Just translate `Firstname Lastname`.